src/javascripts/modules/pills.js

```diff
@@ -0,0 +1,40 @@
+export default class Pills {
+  constructor(el) {
+    this.el = el
+    this.setupListeners()
+  }
+
+  setupListeners() {
+    window.addEventListener('load', () => this.handleLoad())
+  }
+
+  handleLoad() {
+    clearTimeout(this.timeout)
+    this.timeout = setTimeout(() => this.animatePills(), 500)
+    this.releaseSqueezeTimeout = setTimeout(() => this.releasePillSqueeze(), 4500)
+  }
+
+  animatePills() {
+    if (this.el.classList.contains('yellow-pill')) {
+      this.el.classList.add('yellow-pill-roll')
+    } else if (this.el.classList.contains('more-pills')) {
+      this.el.classList.add('more-pills-roll')
+    } else if (this.el.classList.contains('purple-pills')) {
+      this.el.classList.add('purple-pills-roll')
+    } else if(this.el.classList.contains('title-tablet')) {
+      this.el.classList.add('title-tablet-roll')
+    } else if(this.el.classList.contains('title-pill-red')) {
+      this.el.classList.add('title-pill-red-bounce')
+    } else {
+      this.el.classList.add('pills-roll')
+    }
+  }
```

Comment:
this is probably the most direct way to do what you are doing, and i don't think it is wrong, but as you are probably aware, it is extremely repetitive. this conditional essentially does the same thing over and over, just looking for a different name. the way i'd reframe this, since all of your elements will do the same thing, is group all of the images accredited to `data-module="pills"` and place them within a parent div that contains only them so it would look like this:
```html
<div data-module="pills">
  <img src="images/yellow-pill.svg" alt="" class="yellow-pill">
  <img src="images/pills.svg" alt="" class="pills">	  
  <img src="images/pills.svg" alt="" class="pills">
  <img src="images/more-pills.svg" alt="" class="more-pills">	
  <img src="images/more-pills.svg" alt="" class="more-pills">
  <img src="images/purple-pills.svg" alt="" class="purple-pills">
  <img src="images/purple-pills.svg" alt="" class="purple-pills">
</div>
```

note that i took the `data-module` attribute off the applicable images.

then, once the window loads, add an arbitrary animation class, like `pills-animating` to the parent div,

then in your css you can do something like this
```css
.pills-animating {
  .yellow-pill {
    // properties you currently have for .yellow-pill-roll
  }
  // etc for your other pills
}
```

then all you would have to do in your js, is:
```js
animatePills() {
  this.el.add('pills-animating')
}
```